.github/workflows/build-package.yml

```diff
@@ -0,0 +1,132 @@
+    runs-on:
+      #- self-hosted
+      - ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          version: '285.0.0'
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          service_account_email: ${{ secrets.GCP_SA_EMAIL }}
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          export_default_credentials: true
+      - run: gcloud info
+      - name: Set up Go 1.15
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.15.12
+        id: go
+      - name: setup Docker
+        uses: docker-practice/actions-setup-docker@0.0.1
+        with:
+          docker_version: 20.10
+          docker_channel: stable
+      - name: setup Docker Buildx
+        id: buildx
+        uses: crazy-max/ghaction-docker-buildx@v3
+        with:
+          buildx-version: latest
+          qemu-version: latest 
+      - uses: actions/checkout@v2.1.0
+      - uses: jitterbit/get-changed-files@v1
+        id: changed-files
+        with:
+          format: space-delimited
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/checkout@v2
+        with:
+            path: src/github.com/goharbor/harbor-arm
+      - name: Build Base Image
+        if: contains(steps.changed-files.outputs.modified, 'Dockerfile.base') || contains(steps.changed-files.outputs.modified, 'VERSION')
+        run: |
+          set -x
+          echo "BUILD_BASE=true" >> $GITHUB_ENV
+      - name: Build Harbor-ARM Images
+        run: |
+          set -x
+          env
+          # the target release version is the version of next release(RC or GA). It needs to be updated on creating new release branch.
+          target_release_version=$(cat ./VERSION)-arm
+          Harbor_Package_Version=$target_release_version-'build.'$GITHUB_RUN_NUMBER
+
+          if [[ $target_branch == "master" ]]; then
+            Harbor_Assets_Version=$Harbor_Package_Version
+            harbor_target_bucket=$harbor_builds_bucket
+          else
+            Harbor_Assets_Version=$target_release_version
+            harbor_target_bucket=$harbor_releases_bucket/$target_branch
+          fi
+
+          if [[ $target_branch == "release-"* ]]; then
+            Harbor_Build_Base_Tag=$target_release_version
+          else
+            Harbor_Build_Base_Tag=dev-arm
+          fi
+          cd src/github.com/goharbor/harbor-arm
+          sudo make build BASEIMAGETAG=${Harbor_Build_Base_Tag} VERSIONTAG=${Harbor_Assets_Version} PKGVERSIONTAG=${Harbor_Package_Version} BUILDBIN=true NOTARYFLAG=true CHARTFLAG=true TRIVYFLAG=true HTTPPROXY=
+          echo -en "${{ secrets.HARBOR_SIGN_KEY }}" |  gpg --import
+          gpg -v -ab -u ${{ secrets.HARBOR_SIGN_KEY_ID }} $harbor_online_build_bundle
+
+          publishImage $target_branch $Harbor_Assets_Version "${{ secrets.DOCKER_HUB_USERNAME }}" "${{ secrets.DOCKER_HUB_PASSWORD }}"
+      # - name: Build Package
```

Comment:
please enable it for online installer as discussed

Reply:
ok